web-ui/src/main/resources/catalog/components/search/resultsview/partials/viewtemplates/editor.html

```diff
@@ -43,7 +43,7 @@
                 <small data-translate="">updatedOn</small>
                 <small
                   class="text-muted"
-                  data-gn-humanize-time="{{md.dateStamp}}"
+                  data-gn-humanize-time="{{md.changeDate}}"
```

Comment:
Should we also sort by this field in this case to get correct results?
![image](https://github.com/geonetwork/core-geonetwork/assets/1701393/9c7e3748-177e-46df-8e11-2a1e8feedff9)

Reply:
I think so @fxprunayre, I'll update the pull request, thanks.

Reply:
@fxprunayre done in https://github.com/geonetwork/core-geonetwork/pull/7910/commits/36a0d3a6d2dd14e0f35262e6419f06834dccc46b